app/src/main/java/it/niedermann/nextcloud/deck/ui/card/details/CardDetailsFragment.java

```diff
@@ -175,14 +175,18 @@ private void setupDescription() {
             registerEditorListener(binding.descriptionEditor);
             registerEditorListener(binding.descriptionViewer);
         } else {
+            binding.descriptionViewer.setMarkdownString(viewModel.getFullCard().getCard().getDescription());
+
             binding.descriptionEditor.setEnabled(false);
             binding.descriptionEditorWrapper.setVisibility(GONE);
-            binding.descriptionViewer.setEnabled(false);
+            binding.descriptionViewer.setEnabled(true);
```

Comment:
This change is not necessary to view the description, but it seems mandatory to interact with it (links, selection...)